### PR TITLE
Add Asterdio, Mandala and Softhover

### PR DESCRIPTION
**Pull request by bhattaraib58**
Add Following companies:
- Asterdio Inc,
- Mandala Pvt. Ltd.
- Softhover

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Allied Netlink Technology Pvt. Ltd.](http://ant.com.np/) | Putalisadak, Kathmandu | Internet Service Provider (ISP)
 - [Ambarkaar Software Pvt. Ltd.](http://compliancequest.com/) | Bakhundol, Lalitpur | Ambarkaar is a subsidiary company of ComplianceQuest, we develop Enterprise Quality Management System in Salesforce
 - [Anubhabi Technologies Pvt. Ltd.](http://www.anubhabi.com/) | Koteshwor, Kathmandu | At Anubhabi we provide high-end online web and software development solutions to start-ups and medium to large scale companies
+- [Asterdio Inc.](https://asterdio.com/) | Sankhamul, Kathmandu | Web Design/Development, Multimdeia Development, Social Media Management and Hosting Solutions
```
Comment (mesaugat) [inline README.md]:
```diff
- Multimdeia
- Multimedia
```

File: README.md [hunk 2]
```diff
@@ -145,6 +147,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Segonatech](https://segonatech.com/) | Battisputali, Kathmandu | Web Development, Mobile Development, Digital Marketing and Branding, ERP Solutions
 - [Semantic Creation](http://www.semanticcreation.com/) | Pulchowk, Lalitpur | Provide professional services covering all areas of modern technology
 - [Smartmobe Solutions](https://smartmobe.com/) | Tripureshwor, Kathmandu | Mobile Apps, Graphic Design, and Web Development Services
+- [Softhover](https://www.softhover.com/) | Bansbari Rd & Kapan Marg, Kathmandu | Mobile Apps and Web Development Services
```
Comment (mesaugat) [inline README.md]: @bhattaraib58 Can you shorten the description down to `Bansbari Rd, Kathmandu` or `Kapan Marg, Kathmandu`?

Comment (bhattaraib58) [reply README.md]: sure dai

Comment (bhattaraib58) [reply README.md]: Sure dai.

**Pull request merged**